server/android/src/android.rs

```diff
@@ -24,23 +22,6 @@ pub mod android {
 
     static SERVER_BUCKET: Mutex<Option<ServerBucket>> = Mutex::new(None);
 
-    struct AndroidLogger {}
```

Comment:
Not sure the reasoning behind logging twice in standard oms log file and in android logcat... from what I can see, simple_log doesn't have the functionality to log to android logcat 🤔

Reply:
Hmm, can you please specify how we can look at these logs in emulator or on device when running app in debug mode ?

I found logcat logs super useful, and I think we should describe an alternative to inspect logs, when console logs are not available, i.e. in emulator (and possible on physical device), you can go to browse file through Android Studio, device manager, then `/data/user/0/org.openmsupply.client/files/remote_server.log`

Reply:
Do we need android-log crate still ?

Reply:
Yes, will need android-log crate still if we want logcat 🤔